cmd/infracost/comment.go

```diff
@@ -68,10 +68,10 @@ func buildCommentBody(cmd *cobra.Command, ctx *config.RunContext, paths []string
 	if ctx.IsCloudEnabled() && !dryRun {
 		if ctx.Config.IsSelfHosted() {
 			ui.PrintWarning(cmd.ErrOrStderr(), "Infracost Cloud is part of Infracost's hosted services. Contact hello@infracost.io for help.")
+		} else {
+			combined.Metadata.InfracostCommand = "comment"
+			combined.RunID, combined.ShareURL = shareCombinedRun(ctx, combined, inputs)
 		}
-
-		combined.Metadata.InfracostCommand = "comment"
-		combined.RunID, combined.ShareURL = shareCombinedRun(ctx, combined, inputs)
 	}
```

Comment:
yeah that's a fair question.  the changes caused by the refactoring are that the metadata doesn;t contain the 'runIds' value with the list of run ids but I think that's probably ok since we were only setting that on dashboard upload anyway (i.e it wouldn't happen if cloud is disabled either)

Reply:
👍